docs/pages/index.tsx

```diff
@@ -1,47 +1,73 @@
 import type { NextPage } from 'next';
 import Head from 'next/head';
-import Link from 'next/link'
-import styles from '../styles/Home.module.css';
+import Link from 'next/link';
 
 const Home: NextPage = () => {
   return (
-    <div className={styles.container}>
+    <div className="min-w-0 max-w-2xl flex-auto px-4 py-16 lg:max-w-none lg:pr-0 lg:pl-8 xl:px-16">
       <Head>
         <title>CooperTS</title>
         <meta name="description" content="Documentation for CooperTS by ExecOnline" />
       </Head>
 
-      <main className={styles.main}>
-        <h1 className={styles.title}>
+      <main>
+        <p className="lead">
           Welcome to <a href="#">CooperTS!</a>
-        </h1>
+        </p>
 
-        <div className={styles.grid}>
-          <Link href="doc/1">
-            <a href="#" className={styles.card}>
-              <h2>Documentation &rarr;</h2>
-              <p>In-depth information on CooperTS features and API</p>
-            </a>
-          </Link>
+        <div className="not-prose my-12 grid grid-cols-1 gap-6 sm:grid-cols-3">
+          <div className="group relative rounded-xl border border-slate-200 dark:border-slate-800">
+            <div className="absolute -inset-px rounded-xl border-2 border-transparent opacity-0 [background:linear-gradient(var(--link-grid-hover-bg,theme(colors.sky.50)),var(--link-grid-hover-bg,theme(colors.sky.50)))_padding-box,linear-gradient(to_top,theme(colors.indigo.400),theme(colors.cyan.400),theme(colors.sky.500))_border-box] group-hover:opacity-100 dark:[--link-grid-hover-bg:theme(colors.slate.800)]" />
+            <div className="relative overflow-hidden rounded-xl p-6">
+              <h2 className="font-display text-base text-slate-900 dark:text-white">
+                <Link href="doc/1" className="relative overflow-hidden rounded-xl p-6">
+                  <a href="#">
+                    <span className="absolute -inset-px rounded-xl" />
+                    Documentation &rarr;
+                  </a>
+                </Link>
+              </h2>
+              <p className="mt-1 text-sm text-slate-700 dark:text-slate-400">
+                In-depth information on CooperTS features and API
+              </p>
+            </div>
```

Comment:
The right arrow is being read weird by screen readers, and appears on both the headings menu and the links menu. We will need to add an `aria-label` with just the heading text (sans arrow) to both the `<a>` tag and the `<h2>` tag. The subtitle/paragraph of each of these main links should also be given an id to be used as the `aria-describedby` value for the `<a>` tag.
 I also suggest moving the heading to be contained within the link, rather than the other way around. This will allow assistive technology to treat the text as a link first, and a heading second, giving its link functionality priority. 

```suggestion
            <div className="relative overflow-hidden rounded-xl p-6">
              <Link href="doc/1" className="relative overflow-hidden rounded-xl p-6">
                <a href="#" aria-label="Documentation" aria-describedby="doc-desc">
                  <h2 className="font-display text-base text-slate-900 dark:text-white" aria-label="Documentation">
                    <span className="absolute -inset-px rounded-xl" />
                    Documentation &rarr;
                  </h2>
                </a>
              </Link>
              <p className="mt-1 text-sm text-slate-700 dark:text-slate-400" id="doc-desc">
                In-depth information on CooperTS features and API
              </p>
            </div>
```